javascript-frameworks-resources.md

```diff
@@ -91,16 +91,16 @@
     * Lesson 15: Summary of the series (Coming soon...)
 
 ## Meteor
-* [Your First Meteor Application, A Complete Beginner’s Guide to the Meteor JavaScript Framework](http://meteortips.com/book/)
 * [BulletProof Meteor](https://bulletproofmeteor.com/)
+* [Your First Meteor Application, A Complete Beginner’s Guide to the Meteor JavaScript Framework](http://meteortips.com/book/)
 
 ## React
-* [React Tutorial](https://facebook.github.io/react/docs/tutorial.html)
+* [Intro to the React Framework](http://code.tutsplus.com/tutorials/intro-to-the-react-framework--net-35660)
 * [Learning React.js: Getting Started and Concepts](https://scotch.io/tutorials/learning-react-getting-started-and-concepts)
-* [React.js Tutorial: Now is Your Time to Try It, Right in Your Browser](https://codegeekz.com/react-js-tutorial)
-* [React.js tutorial - Creating components](http://ghost.stevenisekimart.in/react-js-tutorial-creating-components)
+* [React Tutorial](https://facebook.github.io/react/docs/tutorial.html)
```

Comment:
@vhf the deleted link [you mentioned](https://github.com/vhf/free-programming-books/pull/1671/files#r43247939) is right here, L100. I thought the same thing, too, at first